packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -0,0 +1,709 @@
+    "id": "aut1nbvIgEenhwE6c0g4",
+    "key": "recovery_question",
+    "status": "ACTIVE",
+    "name": "Security Question",
+    "created": "2020-07-26T21:05:23.000Z",
+    "lastUpdated": "2020-07-26T21:05:23.000Z",
+    "_links": {
+      "self": {
+        "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nbvIgEenhwE6c0g4",
+        "hints": {
+          "allow": [
+            "GET"
+          ]
+        }
+      },
+      "methods": {
+        "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nbvIgEenhwE6c0g4/methods",
+        "hints": {
+          "allow": [
+            "GET"
+          ]
+        }
+      },
+      "deactivate": {
+        "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nbvIgEenhwE6c0g4/lifecycle/deactivate",
+        "hints": {
+          "allow": [
+            "POST"
+          ]
+        }
+      }
+    }
+  }
+]
+```
+
+### Get an Authenticator by id
+
+<ApiOperation method="get" url="/api/v1/authenticators/${authenticatorId}" />
+
+Fetches a Authenticator by its `id`.
+
+#### Request path parameters
+
+| Parameter          | Type   | Description                                            |
+| ------------------ | ------ | ------------------------------------------------------ |
+| `authenticatorId`  | String | The Authenticator's unique identifier.                 |
+
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+N/A
+
+#### Response body
+
+The requested [Authenticator](#Authenticator-object).
+
+#### Usage example
+
+This request returns the Authenticator with an `id` value of `aut1nd8PQhGcQtSxB0g4`:
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4"
+```
+
+##### Response
+
+```json
+{
+    "type": "security_key",
+    "id": "aut1nd8PQhGcQtSxB0g4",
+    "key": "webauthn",
+    "status": "ACTIVE",
+    "name": "Security Key or Biometric",
+    "created": "2020-07-26T21:16:37.000Z",
+    "lastUpdated": "2020-07-27T18:59:30.000Z",
+    "_links": {
+        "self": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "methods": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/methods",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        },
+        "deactivate": {
+            "href": "https://{yourOktaDomain}/api/v1/authenticators/aut1nd8PQhGcQtSxB0g4/lifecycle/deactivate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}
+```
+
+##### Error response example %optional%
```

Comment:
The only possible causes for error when using GET:
1. Invalid API token 
2. AuthenticatorId is invalid or doesn't exist

Let me know if you need example for the 2nd case since it has API specific error message.

Reply:
(2) seems like something worth mentioning. Are there any other new errors?